snakemake_executor_plugin_slurm/__init__.py

```diff
@@ -47,7 +87,8 @@ class ExecutorSettings(ExecutorSettingsBase):
         metadata={
             "help": """
                     Allow requeuing preempted of failed jobs,
-                    if no cluster default. Results in `sbatch ... --requeue ...`
+                    if no cluster default. Results in
+                    `sbatch ... --requeue ...`
                     This flag has no effect, if not set.
                     """,
```

Comment:
let us not use multiline strings for these, but rather simple strings that are auto-concatenated by python if just separated by whitespace.